docs/test-ui-bundle.md

```diff
@@ -0,0 +1,23 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with full content, build as usual the 
+documentation with the antora command but add the option `--ui-bundle-url <path-to-your-local-ui-bundle>/ui-bundle.zip` to your command.
+The location for this bundle is in the `docs-ui`reprository in directory `build/`.
+More details how the create the `ui-bundle.zip` can be found in the `docs-ui`reprository.
+
+The following example assumes:
+
+- You are in the root directory of your local `docs`repository
+- The `docs-ui`repository directory is on the same directory level as `docs`
+- You have `ui-bundle.zip` manually created with `gulp pack` (for details see `docs-ui`)
```

Comment:
Why not just link "with `gulp pack`" to the relevant section of the documentation (and remove the rest of the sentence)?